.travis.yml

```diff
@@ -1,5 +1,2 @@
 language: node_js
 node_js: 6.9.1 # latest LTS
-cache:
-  directories:
-    - node_modules/phantomjs-prebuilt
```

Comment:
We can cache the `node_modules/` directory.

https://docs.travis-ci.com/user/languages/javascript-with-nodejs/#Caching-with-npm